src/components/Header/Header.tsx

```diff
@@ -39,23 +48,39 @@ export const Header = (): ReactElement => {
             />
           </Link>
         </div>
-        <div className="hero__nav hero__nav--hidden -mt-0.5 w-full max-w-full">
-          <nav className="main-nav inset-0 flex items-center">
-            <ul className="main-nav__list flex m-0">
-              {mainNavItems.map(({ title, href }) => {
-                return (
-                  <li key={title} className="main-nav__item float-left text-center">
-                    <Link
-                      href={href}
-                      className="main-nav__link font-rex mr-5 text-[14.6px] hover:text-orange-400 transition delay-150"
-                    >
-                      {title}
-                    </Link>
-                  </li>
-                );
-              })}
-            </ul>
-          </nav>
+        <div className="hero__nav hero__nav--hidden w-full max-w-full">
+          <div
+            onClick={() => {
+              setOpen(!open);
```

Comment:
you should use a callback when creating a state based on the prev state https://ogzhanolguncu.com/blog/how-does-prevstate-works-under-the-hood